code-orange/LocationsProvider.swift

```diff
@@ -35,16 +36,39 @@ class LocationsProvider {
     let coronaNotMatchingLocations = coronaLocations.filter { !infectedMatchedLocations.contains($0) }
     return Locations(matchedLocations: matchingLocations, otherLocations: coronaNotMatchingLocations)
   }
-  
+
   private func getStoredCoronaLocations() -> [RecordedLocation]? {
     // this is temp
     let communicator = Communicator()
     return communicator.getServerResults()
   }
```

Comment:
we need to change this func to be named "getCoronaLocations"
it should load the stored corona locations, and query the server for new locations (since last querying time)
if you add the storing in this pr then the loading should be done here

Reply:
I would address this in the server API PR